Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM aeternity/builder:1804 as pluginbuild
+ADD rebar.config rebar.lock rebar.config.script /aeplugin_dev_mode/
+ADD src /aeplugin_dev_mode/src/
+ADD priv /aeplugin_dev_mode/priv/
+
+RUN cd /aeplugin_dev_mode && rebar3 compile
+
+FROM aeternity/aeternity:master
+
+COPY --from=pluginbuild /aeplugin_dev_mode/_build/default/lib/aeplugin_dev_mode /home/aeternity/node/plugins/aeplugin_dev_mode
+
+# I have no idea why this extra step is needed
+# But without it, the JSON-Schema file in priv isn't copied
+COPY --from=pluginbuild /aeplugin_dev_mode/_build/default/lib/aeplugin_dev_mode/priv /home/aeternity/node/plugins/aeplugin_dev_mode/priv
+
+COPY ./examples/devmode.yaml /home/aeternity/node/devmode.yaml
```

Comment:
you should be able to get rid of the exmaple config file and use the ENV vars `AE__SYSTEM__PLUGINS` or just `AE__SYSTEM__DEV_MODE` perhaps

Reply:
Yes, you can, but esp. if you actually want to modify the plugin config parameters, or other node parameters, a config file will be more convenient.

Reply:
But in this particular case, it's not needed per se.
Although when I tested it earlier today, I discovered that I had cleaned it too much, and accidentally removed even the `dev_mode: true`. :)